sdc_services/api/fhir.py

```diff
@@ -0,0 +1,22 @@
+
+from flask import Blueprint, request
+
+from sdc_services.models.r2.questionnaire_response import QuestionnaireResponse
+from sdc_services.models.r2.bundle import as_bundle
+
+R2_PREFIX = '/v/r2/fhir'
```

Comment:
FHIR versions, grr ... didn't you tell me we're using R3/STU3 for Questionnaire and friends?

Reply:
We're a bit...scattered. DSTU2 (r2) for QuestionnaireResponse and Observation (if I recall correctly), STU3 (r3) for Questionnaire